src/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/CryptographicUnexpectedOperationException.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+
+namespace System.Security.Cryptography
+{
+    public class CryptographicUnexpectedOperationException : CryptographicException
+    {
+        public CryptographicUnexpectedOperationException()
+            : base(SR.Arg_CryptographyException)
+        {
+        }
```

Comment:
That's fair.  @stephentoub any thoughts on if we care about making the HResult values of the default ctors of these two exceptions match desktop?  The HResult value of CryptographicException is pretty hard to depend on since we don't try to normalize the codes between Windows and Unix (or across versions of Windows).